include/util/range_table.hpp

```diff
@@ -79,9 +79,9 @@ template <unsigned BLOCK_SIZE, storage::Ownership Ownership> class RangeTable
         unsigned last_length = 0;
         unsigned lengths_prefix_sum = 0;
         unsigned block_idx = 0;
-        unsigned block_counter = 0;
         BlockT block;
 #ifndef BOOST_ASSERT_IS_VOID
+        unsigned block_counter = 0;
```

Comment:
These changes were merged to master in another PR https://github.com/Project-OSRM/osrm-backend/pull/6596 - they should not also be showing up in _this_ PR.  I suspect some merges have been incorrectly applied or lost on the branch.  This is probably also why CI is failing.

Reply:
The source code on master branch has `block_counter = 0` on line 85, meanwhile in the commit it is placed on line 84. Could that be what's causing CI to fail?

Reply:
@mugr1x may be try to squash all your commits into single one? Or open another PR? Tbh for me it is not clear why it is happening - looks more like some GH bug.